pkg/agent/route/route_linux.go

```diff
@@ -863,6 +968,97 @@ func (c *Client) initServiceIPRoutes() error {
 	return nil
 }
 
+func (c *Client) initNodeNetworkPolicy() {
+	c.fixedNodeNetworkPolicyChains = []string{
+		antreaInputChain,
+		antreaOutputChain,
+		privilegedNodeNetworkPolicyIngressRulesChain,
+		privilegedNodeNetworkPolicyEgressRulesChain,
+		config.NodeNetworkPolicyIngressRulesChain,
+		config.NodeNetworkPolicyEgressRulesChain,
+	}
+	c.buildFixedNodeNetworkPolicyIPTablesRules(c.networkConfig.IPv4Enabled, c.networkConfig.IPv6Enabled)
```

Comment:
Maybe name it `buildStaticNodeNetworkPolicyIPTablesRules`

Reply:
Done